plugin.yml

```diff
@@ -1,14 +1,14 @@
-name: create-ecr
-description: Plugin managing creation of AWS ECR
+name: Create ECR
```

Comment:
Title case is recommended per https://github.com/seek-oss/aws-sm-buildkite-plugin/pull/8.